modules/authorization/lock/README.md

```diff
@@ -61,7 +61,7 @@ The following module usage examples are retrieved from the content of the files
 <summary>via Bicep module</summary>
 
 ```bicep
-module locks './authorization/lock/main.bicep' = {
+module lock './authorization/lock/main.bicep' = {
```

Comment:
Lock vs locks?

Reply:
Not sure if I got this right. Are you suggesting to keep the locks module name plural and clarify it later as done for tags?

Reply:
I am wondering if there was an update on this and if we should change it now to potentially change it back later.

Reply:
I don't know what to say. I wasn't either and won't be for a while 😆 so either we proceed or we put this whole renaming on hold until we address this topic. I just fear in the latter case we may end up blocking other prs to main or worse piling up conflicts to this bigger change

Reply:
Hm ok, in